public/request/comment/create.php

```diff
@@ -20,6 +20,7 @@
 $articleType = requestInputPost('t', null, 'integer');
 
 $commentPayload = requestInputPost('c');
+// printable characters only. see https://www.asciitable.com/
```

Comment:
Probably something for a separate PR, but is there some reason we shouldn't support non-ascii characters, or at least replace them with a question mark instead of stripping them entirely?

Reply:
It'll probably mess with the character counting as non-ASCII UTF-8 characters take more than 1 latin1 character each. But we've started allowing UTF-8 in other places, so it seems reasonable to allow it here. As long as users don't try to completely fill the field with non-ASCII characters, it should be okay.

Reply:
Agreed, let's have that. We already allow it in forum comments where the counter is thrown off by the same issue 👍